components/form/ErrorMessage.js

```diff
@@ -1,22 +1,24 @@
 import { useFormContext } from 'react-hook-form';
 import PropTypes from 'prop-types';
-export function ErrorMessage({ errorClassName, name }) {
+export function ErrorMessage({ className, name }) {
   const {
     formState: { errors }
   } = useFormContext();
 
   return (
     <>
-      {errors?.[name] && <div className={`text-amcovad-danger py-1 ${errorClassName} `}>{errors?.[name]?.message}</div>}
+      {errors?.[name] && (
+        <div className={`text-amcovad-danger py-1 text-sm ${className} `}>{errors?.[name]?.message}</div>
+      )}
     </>
   );
 }
 ErrorMessage.propTypes = {
   name: PropTypes.string,
-  errorClassName: PropTypes.string
+  className: PropTypes.string
 };
 
 ErrorMessage.defaultProps = {
   name: null,
-  errorClassName: null
+  className: null
```

Comment:
Even though it is not important, it is best practises to always arrange the prop types alphabetically.

```js
ErrorMessage.propTypes = {
  className: PropTypes.string,
  name: PropTypes.string
};

ErrorMessage.defaultProps = {
  className: null,
  name: null
}
```